homework_2/fastq_filtrator.py

```diff
@@ -0,0 +1,91 @@
+# (╮°-°)╮┳━━┳ ( ╯°□°)╯ ┻━━┻ (╮°-°)╮┳━━┳ ( ╯°□°)╯ ┻━━┻ (╮°-°)╮┳━━┳ ( ╯°□°)╯ ┻━━┻
+
+def length_check(sequence, length_bounds):
+    """
+    Checking if length of a sequence is inside bounds
+    """
+    if type(length_bounds) is tuple:
+        min, max = length_bounds
+    else:
+        min, max = 0, length_bounds
+    if len(sequence) >= min and len(sequence) <= max:
+        return True
```

Comment:
Логично было бы увидеть возвращаемые значения `True` и `False`, а не `True` и `None`
```suggestion
    return min <= len(sequence) <= max:
```